front/src/Components/AuthentificationForm.tsx

```diff
@@ -0,0 +1,68 @@
+import * as React from 'react';
+import Button from '@mui/material/Button';
+import TextField from '@mui/material/TextField';
+import ParameterCardTitle from './ParameterCard/ParameterCardTitle';
+import Grid from '@mui/material/Grid/Grid';
+import { Link } from "react-router-dom";
+import { useFormManager } from "react-simple-form-manager";
+
+interface AuthentificationFormState {
+  username: string;
+  password: string;
+}
+
+const minimumFieldLength = 8;
+
+const fieldValidator = (field: string) => {
+  return field.length < minimumFieldLength;
+}
+const errorMessage = "Must be at least 8 characters long";
+
+
+const fieldHasError = (fieldValue: string) => {
+  if (!fieldValue)
+    return false;
+  return fieldValidator(fieldValue);
+}
+
+const AuthentificationForm = (props: any) => {
```

Comment:
When asking the props, you can destructure them inside the call instead of props:any
An exemple here would be
```

interface AuthentificationFormProps = {
   signup: bool,
   onSubmit: // function type I don t know this function declaration
   debug: bool?,
}

const AuthentificationForm = ({signup, onSubmit, debug} : AuthenficationFormProps) => {
```
you can these use them as signup and onSubmit instead of props.signup with props as type any

Reply:
Good to know. Fixed